cmd/podman/down/down.go

```diff
@@ -0,0 +1,23 @@
+package down
+
+import (
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Command: podman _down_
+	downCmd = &cobra.Command{
```

Comment:
I tested this manually using an example I found in the docs - here were my contents of play.yaml:

```
# Save the output of this file and use kubectl create -f to import
# it into Kubernetes.
#
# Created with podman-0.11.2-dev
apiVersion: v1
kind: Pod
metadata:
  creationTimestamp: 2018-12-03T19:07:59Z
  labels:
    app: some-mariadb
  name: some-mariadb-libpod
spec:
  containers:
  - command:
    - docker-entrypoint.sh
    - mysqld
    env:
    - name: HOSTNAME
    - name: GOSU_VERSION
      value: "1.10"
    - name: GPG_KEYS
      value: "199369E5404BD5FC7D2FE43BCBCB082A1BB943DB \t177F4010FE56CA3336300305F1656F24C74CD1D8
        \t430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \t4D1BB29D63D98E422B2113B19334A25F8507EFA5"
    - name: MARIADB_MAJOR
      value: "10.3"
    - name: MARIADB_VERSION
      value: 1:10.3.10+maria~bionic
    - name: MYSQL_ROOT_PASSWORD
      value: x
    image: quay.io/baude/demodb:latest
    name: some-mariadb
    ports:
    - containerPort: 3306
      hostPort: 36533
    resources: {}
    securityContext:
      capabilities:
        drop:
        - CAP_MKNOD
        - CAP_NET_RAW
        - CAP_AUDIT_WRITE
    tty: true
status: {}
```

Things don't actually seem to start, but the output looks reasonable?

```
0:24 jawnsy@pattern ~/projects/podman $ sudo ./bin/podman play kube play.yaml 
WARN[0000] CNI config list /etc/cni/net.d/87-podman-ptp.conflist has the same network name as /etc/cni/net.d/87-podman-bridge.conflist, skipping 
[error starting container 1472f447e091ade9c6c2ac1fec4c1c25892a67ebc47c06c0d317f1f1172d9e80: container create failed (no logs from conmon): EOF]
Pod:
b94f21bf8fca167279e09e29ce55c8245d4c2d6cf16d79f92d1bf3da9410545c
Container:
1472f447e091ade9c6c2ac1fec4c1c25892a67ebc47c06c0d317f1f1172d9e80

error starting container 1472f447e091ade9c6c2ac1fec4c1c25892a67ebc47c06c0d317f1f1172d9e80: container create failed (no logs from conmon): EOF

Error: failed to start 1 containers
20:24 jawnsy@pattern ~/projects/podman $ sudo ./bin/podman down kube play.yaml 
Pods stopped:
b94f21bf8fca167279e09e29ce55c8245d4c2d6cf16d79f92d1bf3da9410545c
Pods removed:
b94f21bf8fca167279e09e29ce55c8245d4c2d6cf16d79f92d1bf3da9410545c
```

I'm running as root here because of something to do with `rootlessport`; I guess I'm missing a config:

```
error starting container 892779fa08819486ff2642f60ae8c581ec0d29da8174191db701897be03ff8a0: could not find "rootlessport" in one of [/usr/local/libexec/podman /usr/local/lib/podman /usr/libexec/podman /usr/lib/podman].  To resolve this error, set the helper_binaries_dir key in the `[engine]` section of containers.conf to the directory containing your helper binaries.
```